docs/compass/06-01-runtime-agent-compass-connection.md

```diff
@@ -0,0 +1,73 @@
+---
+title: CompassConnection
+type: Custom Resource
+---
+
+The `compassconnections.compass.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to prevent the status of the connection between Runtime Agent and Compass. The `CompassConnection` custom resource (CR) contains the connection statuses and Compass URLs. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
`...used to prevent the status of the connection between Runtime Agent and Compass.` - not sure if prevent is the correct verb here.

Reply:
🙈 Right so! It was supposed to be `preserve`.